### DISCUSSION TITLE
Where to find documentation on plots

### DISCUSSION THREAD

**Discussion by nprime496** [pycaret/pycaret#2870] [Q&A]
Hi, how to know what plots are available from [plot_model](https://pycaret.gitbook.io/docs/get-started/release-notes#plot_model) ? Is there an exhaustive list somewhere ?

edit: I found this [site](https://www.projectpro.io/recipes/what-are-stack-models-plot-model-and-evaluate-model-functions-classification-model-pycaret#mcetoc_1g4g9tv4r9) giving a list, but is there an official one ?

**Comment by acartro** [comment]
Good,
you can see the plots that can be made with pycaret on this website https://pycaret.readthedocs.io/en/latest/api/time_series.html by checking the plot_model function.

The plots that can currently be extracted with time series are:

- ‘ts’ - Time Series Plot

- ‘train_test_split’ - Train Test Split

- ‘cv’ - Cross Validation

- ‘acf’ - Auto Correlation (ACF)

- ‘pacf’ - Partial Auto Correlation (PACF)

- ‘decomp’ - Classical Decomposition

- ‘decomp_stl’ - STL Decomposition

- ‘diagnostics’ - Diagnostics Plot

- ‘diff’ - Difference Plot

- ‘periodogram’ - Frequency Components (Periodogram)

- ‘fft’ - Frequency Components (FFT)

- ‘ccf’ - Cross Correlation (CCF)

- ‘forecast’ - “Out-of-Sample” Forecast Plot

- ‘insample’ - “In-Sample” Forecast Plot

- ‘residuals’ - Residuals Plot

**Reply by ngupta23** [reply]
You can also check out the doc string of the plot_model function for more details about the supported plots.
